src/components/inset-text/inset-text.njk

```diff
@@ -1,7 +1,5 @@
 {% from "inset-text/macro.njk" import govukInsetText %}
 
 {{ govukInsetText({
-  classes: '',
-  html: 'It can take up to 8 weeks <a href="#">to register</a> a lasting power of attorney if there are no mistakes in the application.'
-  })
-}}
+    html: 'It can take up to 8 weeks <a href="#">to register</a> a lasting power of attorney if there are no mistakes in the application.'
+}) }}
```

Comment:
That'd then require escaping the double quoted href attribute – personally I'd prefer to use single quotes rather than escaping in those scenarios, but that's a loosely held preference 🙂

Reply:
I think in this specific circumstance single quotes is reasonable given we want to keep the HTML the same as elsewhere.

Reply:
👍